toxcore/group.c

```diff
@@ -141,33 +141,28 @@ static int32_t create_group_chat(Group_Chats *g_c)
     return id;
 }
 
-
-/* Wipe a groupchat.
- *
- * return -1 on failure.
- * return 0 on success.
+/*
```

Comment:
I've been changing comments to doxygen style, so starting with `/**` and with `@return` and `@param`.